vault/logical_system.go

```diff
@@ -821,6 +821,27 @@ func NewSystemBackend(core *Core) *SystemBackend {
 				HelpSynopsis:    strings.TrimSpace(sysHelp["plugin-catalog"][0]),
 				HelpDescription: strings.TrimSpace(sysHelp["plugin-catalog"][1]),
 			},
+			&framework.Path{
+				Pattern: "plugin/backend/reload$",
```

Comment:
Should this be `plugins/` to match the existing endpoint? Also since it operates on multiple plugins